tests/components/modbus/test_init.py

```diff
@@ -63,13 +79,54 @@
 from homeassistant.setup import async_setup_component
 import homeassistant.util.dt as dt_util
 
-from .conftest import TEST_MODBUS_NAME, ReadResult
-
 from tests.common import async_fire_time_changed
 
 TEST_SENSOR_NAME = "testSensor"
 TEST_ENTITY_ID = f"{SENSOR_DOMAIN}.{TEST_SENSOR_NAME}"
 TEST_HOST = "modbusTestHost"
+TEST_MODBUS_NAME = "modbusTest"
+
+
+class ReadResult:
+    """Storage class for register read results."""
+
+    def __init__(self, register_words):
+        """Init."""
+        # result read calls
+        self.registers = register_words
+        self.bits = register_words
+
+        # result write calls
+        self.value = register_words
+        self.count = len(register_words)
+
+
+@pytest.fixture
+def mock_pymodbus():
```

Comment:
Looks like we have the same fixture in conftest.py. Why do we not use that one?

Reply:
Removed. The reason was that I wanted them (and still do) to be different, but that is for a another PR: